.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
   "cSpell.import": ["cspell.yaml"],
-  "go.testFlags": ["-timeout", "5m"]
+  "go.testFlags": ["-timeout", "60m"]
```

Comment:
😑 Why do test take such long time?? Can we NOT run the slow tests by default?

I would love to be able to "run all CLI tests" to verify changes that I am making without waiting for an hour for results. Ideally the whole default test run should complete within a minute or so.

Reply:
Added this change by accident - this was for running functional tests locally.